### ISSUE TITLE
Overfitting traning data. 

### ISSUE THREAD

**Issue by rudeshigrandson** [iArunava/ENet-Real-Time-Semantic-Segmentation#6]
I was training the model with cityscapes dataset with 28 classes.
Training images - 2975
Testing - 500  
Epoch - 300 (broke at 62 as not getting empirical data to go ahead)

mIOU for various epochs are as follows.

1st Epoch - 32.65
7th Epoch - 34.06
15th Epoch - 13.81
25th Epoch - 12.68
35th Epoch - 12.91
45th Epoch - 17.25
55th Epoch - 20.57
60th Epoch - 17.36
62th Epoch - 20.57

When I tried to run the inference on the model was clearly able to see that it was trying to overfit the training image. I was continuously able to see the structure of the Mercedes logo.

**Comment by AvivSham** [comment]
Did you solve the problem?  if not can you give more details? setup and what you tired so far?

**Comment by kai3316** [comment]
I have the same problem, could you help me to solve this?

**Comment by AvivSham** [comment]
can you share you validation/train loss graphs throughout training?
